src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java

```diff
@@ -577,187 +546,59 @@ public boolean saveEntitySetToElasticsearch(
-                .setFetchSource( new String[] { ENTITY_SET, PROPERTY_TYPES }, null )
-                .setFrom( start )
-                .setSize( maxHits )
-                .execute()
-                .actionGet();
-
-        List<Map<String, Object>> hits = Lists.newArrayList();
-        response.getHits().forEach( hit -> hits.add( hit.getSourceAsMap() ) );
-        return new SearchResult( response.getHits().getTotalHits(), hits );
-    }
-
-    @Override
-    public boolean updatePropertyTypesInEntitySet( UUID entitySetId, List<PropertyType> updatedPropertyTypes ) {
-        if ( !verifyElasticsearchConnection() ) { return false; }
-
-        Map<String, Object> propertyTypes = Maps.newHashMap();
-        propertyTypes.put( PROPERTY_TYPES, updatedPropertyTypes );
-        try {
-            String s = ObjectMappers.getJsonMapper().writeValueAsString( propertyTypes );
-            UpdateRequest updateRequest = new UpdateRequest(
-                    ENTITY_SET_DATA_MODEL,
-                    ENTITY_SET_TYPE,
-                    entitySetId.toString() ).doc( s, XContentType.JSON );
-            client.update( updateRequest ).actionGet();
-            return true;
-        } catch ( IOException e ) {
-            logger.debug( "error updating property types of entity set in elasticsearch" );
-        }
-        return false;
-    }
-
     /**
      * Add new mappings to existing index.
      * Updating the entity set model is handled in {@link #updatePropertyTypesInEntitySet(UUID, List)}
      *
-     * @param entitySetId        the id of the entity set to which the new proprties are added
-     * @param newPropertyTypes   the ids of the new properties
-     * @param linkedEntitySetIds if it's a linking entity set, the ids of the linked entity sets, otherwise empty
+     * @param entityType       the entity type to which the new properties are added
+     * @param newPropertyTypes the ids of the new properties
      */
     @Override
-    public boolean addPropertyTypesToEntitySet(
-            UUID entitySetId,
-            List<PropertyType> newPropertyTypes,
-            Optional<Set<UUID>> linkedEntitySetIds ) {
-        return addMappingToSecurableObjectIndex( entitySetId, newPropertyTypes, linkedEntitySetIds );
+    public boolean addPropertyTypesToEntityType( EntityType entityType, List<PropertyType> newPropertyTypes ) {
+        saveObjectToElasticsearch( ENTITY_TYPE_INDEX, ENTITY_TYPE, entityType, entityType.getId().toString() );
+        return addMappingToEntityTypeDataIndex( entityType, newPropertyTypes );
     }
 
     @Override
     public boolean addLinkedEntitySetsToEntitySet(
             UUID linkingEntitySetId,
             List<PropertyType> propertyTypes,
             Set<UUID> newLinkedEntitySets ) {
-        return addMappingToSecurableObjectIndex( linkingEntitySetId,
-                propertyTypes,
-                Optional.of( newLinkedEntitySets ) );
+        // TODO do we need this anymore?
```

Comment:
I think we still need to add those properties to document, since with linked entities we store it like this:
<entity_set_id>.<property_type_id>